packages/runtime/container-runtime/package.json

```diff
@@ -184,6 +184,7 @@
 		"@tylerbu/sorted-btree-es6": "^1.8.0",
 		"double-ended-queue": "^2.1.0-0",
 		"lz4js": "^0.2.0",
+		"semver": "^7.5.3",
```

Comment:
why such an old version? this was published 2 years ago. Seems like we might as well grab 7.7.1 (2 months old) while we're adding the dep. I care partially as semver is a common culprit for regex denial of service vulnerability reports. In the past they have never really applied to us, but with some more recent scenarios around running Fluid client packages in a service it's *slightly* more plausible they could (not really a whole ton, since the version specified would still not be user-driven input but probably coming from code, but still)

Reply:
I just used 7.5.3 since that's what we use in other client packages and syncpack would get annoyed. I could update the other packages to 7.71, but maybe that would be better in a separate PR?

Reply:
PR to update semver deps - https://github.com/microsoft/FluidFramework/pull/24248